cosmos/profiles/base.py

```diff
@@ -24,6 +25,66 @@
 logger = get_logger(__name__)
 
 
+@dataclasses.dataclass
+class DbtConfigVars:
```

Comment:
This is great! Can this class instead be named `DbtProfileConfigVars`, so there is no confusion with other dbt [configs](https://docs.getdbt.com/reference/dbt-jinja-functions/config).

Reply:
fix